cmd/di.go

```diff
@@ -306,8 +307,10 @@ func newSessionManagerFactory(
 ) session.ManagerFactory {
 	return func(dialog communication.Dialog) *session.Manager {
 		providerBalanceTrackerFactory := func(consumerID, receiverID, issuerID identity.Identity) (session.BalanceTracker, error) {
-			// if the flag ain't set, just return a noop balance tracker
-			if !nodeOptions.ExperimentPayments {
+			// We want backwards compatibility for openvpn on desktop providers, so no payments for them.
+			// Splitting this as a separate case just for that reason.
+			// TODO: remove this one day.
+			if proposal.ServiceType == openvpn.ServiceType {
```

Comment:
Proposal has `paymentType` - field specially for feature toggle like this:
- `openvpn` services can announce `paymentType=none`
- `noop` & `wireguard` services can announce `paymentType=per_time`

In such way, both Provider & Consumer can explicitly know if ping-pong game should be played

Reply:
Tried implementing this - there's an issue. Old consumers won't be able to decode the new paymentType, therefore they won't see the nodes. So this is a no-go for us. Leaving the same check here for backwards compatibility.